scalafix/build.sbt

```
@@ -8,7 +8,7 @@ lazy val scala3Version = "3.3.1"
 inThisBuild(
   List(
     organization := "bondlink",
-    version := "0.4.0",
+    version := "0.4.1",
```

Let's do a new minor version

```suggestion
    version := "0.5.0",
```